src/einsteinpy/coordinates/core.py

```diff
@@ -30,13 +30,22 @@ def __init__(self, x, y, z):
         self.z = z
         super().__init__(x.si.value, y.si.value, z.si.value)
         self.system = "Cartesian"
+        self._dimension = {
+            "x": self.x,
+            "y": self.y,
+            "z": self.z,
+            "system": "Cartesian",
+        }
 
     def __repr__(self):
         return "Cartesian x: {}, y: {}, z: {}".format(self.x, self.y, self.z)
 
     def __str__(self):
         return self.__repr__()
 
+    def __getitem__(self, item):
+        return self._dimension[item]
```

Comment:
And a change here,
```suggestion
        if isinstance(item, (int, np.integer)):
            return self._dimension[self._dimension_order[item]]
        return self._dimension[item]
```

Does this two comments address this https://github.com/einsteinpy/einsteinpy/pull/387#issuecomment-578191030 ?

Reply:
Also, please write a docstring for `__getitem__`s. Users should know that objects are subscriptable.